distribution/bin/seata-server.bat

```diff
@@ -91,7 +91,7 @@ if exist %BASEDIR%/logs (
   md "%BASEDIR%/logs"
 )
 
```

Comment:
if "%SKYWALKING_ENABLE%"=="true"

Reply:
fix, but without Window environment, to verify correctness